### DISCUSSION TITLE
Showcase your Jaksel Project

### DISCUSSION THREAD

**Discussion by RioChndr** [RioChndr/jaksel-language#34] [Show and tell]
Showcase your awesome little project using jaksel programming language ✨✨

**Comment by idjoo** [comment]
kalo syntax highlighting plugin untuk vim masuk kesini ga bang?

![20-Apr-2022_21-29](https://user-images.githubusercontent.com/45854668/164254045-f83cf369-2613-4d06-a9bc-6b5ada444317.png)

**Reply by RioChndr** [reply]
Woah, Love it. Kreatif banget. nanti buat repo mu sendiri. nanti mention dari Readme project ini aja

**Reply by idjoo** [reply]
Terima kasih banyak bang, nanti aku buat repo terus aku pr

**Reply by RioChndr** [reply]
Sip, ditunggu karyanya
